types/expo/tslint.json

```diff
@@ -2,6 +2,8 @@
     "extends": "dtslint/dt.json",
     "rules": {
         "void-return": false,
-        "max-line-length": false
+        "max-line-length": false,
+        "file-name-casing": false,
```

Comment:
Why was it necessary to disable this?

Reply:
Because the name of the of the tests file is `expo-tests.tsx` which fails the lint.

Reply:
This didn't fail before though, and the file name hasn't changed. This rule isn't in tslint [latest](https://github.com/palantir/tslint/blob/master/src/configs/latest.ts) so I suspect you are somehow using a different config than this one. Try removing the tslint.json changes and seeing if travis fails.